### PR TITLE
sidebar-style-complete

### PR DESCRIPTION
**Pull request by dimahalaiskiy**
https://user-images.githubusercontent.com/74025448/128919839-0478959a-d176-433a-bda5-9467b7afcd4a.mp4

### CODE REVIEW

File: src/assets/icons.svg [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<svg aria-hidden="true" style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (RomPavel) [inline src/assets/icons.svg:1]: lets remove this file

File: src/constants/route-constant.ts [hunk 2]
```diff
@@ -18,6 +18,12 @@ export enum SidebarMenu {
   editor = 'Editor',
   manage = 'Manage',
   statistics = 'Statistics',
+  homeicon = 'home',
```
Comment (RomPavel) [inline src/constants/route-constant.ts:21]: we need separate **enum** for that info

File: src/app/components/sidebar/sidebar.component.ts [hunk 3]
```diff
@@ -12,12 +13,20 @@ import { AuthStoreService } from '../../services/store/auth-store.service';
 export class SidebarComponent implements OnInit {
   isOpen = true;
 
+  isShown = true;
```
Comment (RomPavel) [inline src/app/components/sidebar/sidebar.component.ts:16]: we already have isOpen, why we need isShown

**Comment by RomPavel** [comment]
why open/close arrow icon is reversed? it was more intuitive previously

File: src/app/components/sidebar/sidebar.component.ts (hunk 3 above)
Comment (dimahalaiskiy) [reply src/app/components/sidebar/sidebar.component.ts:16]: I was create this for hide the text in sidebar. But if isOpen can do this too, I will delete isShown.

File: src/constants/route-constant.ts (hunk 2 above)
Comment (dimahalaiskiy) [reply src/constants/route-constant.ts:21]: I will do this after 00:00 pm today

File: src/assets/icons.svg (hunk 1 above)
Comment (dimahalaiskiy) [reply src/assets/icons.svg:1]: already yet

**Comment by Inunor** [comment]
Any updates?

File: src/app/components/sidebar/sidebar.component.ts (hunk 3 above)
Comment (dimahalaiskiy) [reply src/app/components/sidebar/sidebar.component.ts:16]: Changes done

File: src/constants/route-constant.ts (hunk 2 above)
Comment (dimahalaiskiy) [reply src/constants/route-constant.ts:21]: done, sorry for delay

File: src/assets/icons.svg (hunk 1 above)
Comment (RomPavel) [reply src/assets/icons.svg:1]: ?

File: src/app/components/sidebar/sidebar.component.html [hunk 4]
```diff
@@ -1,31 +1,57 @@
...
+            </div>
+            <span class="sidebar-text">Exit</span>
           </mat-list-item>
         </nav>
       </ng-container>
     </mat-nav-list>
   </mat-sidenav>
   <mat-sidenav-content>
     <mat-icon
+      ng-click="toggleSideBar=!toggleSideBar"
```
Comment (RomPavel) [inline src/app/components/sidebar/sidebar.component.html:50]: remove this if its not needed

File: src/app/components/sidebar/sidebar.component.html [hunk 5]
```diff
@@ -1,31 +1,57 @@
 <mat-sidenav-container class="sidebar" autosize *ngIf="{ isSingIn: isSingIn$ | async } as $">
-  <mat-sidenav *ngIf="$.isSingIn" class="sidebar__container" mode="side" [opened]="isOpen">
+  <mat-sidenav
+    *ngIf="$.isSingIn"
+    class="sidebar__container"
+    [style]="getWidth()"
+    mode="side"
+    ng-show="toggleSideBar"
```
Comment (RomPavel) [inline src/app/components/sidebar/sidebar.component.html:7]: remove this if its not needed

File: src/app/components/sidebar/sidebar.component.ts [hunk 6]
```diff
@@ -18,14 +19,20 @@ export class SidebarComponent implements OnInit {
 
   usersRoute: UserRoutesType = usersRoute;
 
+  mode = new FormControl('over');
```
Comment (RomPavel) [inline src/app/components/sidebar/sidebar.component.ts:22]: is it needed?

File: src/app/components/sidebar/sidebar.component.html [hunk 7]
```diff
@@ -1,31 +1,57 @@
 <mat-sidenav-container class="sidebar" autosize *ngIf="{ isSingIn: isSingIn$ | async } as $">
-  <mat-sidenav *ngIf="$.isSingIn" class="sidebar__container" mode="side" [opened]="isOpen">
+  <mat-sidenav
+    *ngIf="$.isSingIn"
+    class="sidebar__container"
+    [style]="getWidth()"
+    mode="side"
```
Comment (RomPavel) [inline src/app/components/sidebar/sidebar.component.html:6]: is it needed?

File: src/app/components/sidebar/sidebar.component.html (hunk 5 above)
Comment (dimahalaiskiy) [reply src/app/components/sidebar/sidebar.component.html:7]: this all exept ng-show="toggleSideBar" is needed. ng-show was removed

File: src/app/components/sidebar/sidebar.component.ts (hunk 6 above)
Comment (dimahalaiskiy) [reply src/app/components/sidebar/sidebar.component.ts:22]: No isnt, I was remove this one

File: src/constants/route-constant.ts [hunk 8]
```diff
@@ -0,0 +1,33 @@
+export enum Route {
```
Comment (RomPavel) [inline src/constants/route-constant.ts:1]: this whole file should not exist

**Pull request merged**